classification_service/main.py

```diff
@@ -59,17 +59,22 @@ def process_event(self, event_type, data):
 
         url = f"{self.model_url}/predict_{event_type}"
 
+        response = None
         with self.request_latency.time():
             try:
                 response = requests.post(
                     url,
                     data=data_json,
                     headers={"Content-Type": "application/json"},
-                    timeout=10
+                    timeout=20
                 )
             except requests.Timeout:
                 print("The request timed out")
```

Comment:
suggestion: I think you want to also use logging here and never use print, if you already use logging anyways. Otherwise if the log config ever changes to log somewhere else, only part of the information will actually end up there.

Reply:
Thanks yeah I think that was left over from debugging!